src/content-feature.js

```diff
@@ -235,6 +235,24 @@ export default class ContentFeature {
     load (args) {
     }
 
+    /**
+     * This is a wrapper to ensure all calls are typed via schemas.
+     * @type {(...args: any[]) => any}
+     */
+    notify (...args) {
+        const [name, params] = args
+        this.messaging.notify(name, params)
+    }
+
+    /**
+     * This is a wrapper to ensure all calls are typed via schemas.
+     * @type {(...args: any[]) => Promise<any>}
+     */
+    request (...args) {
+        const [name, params] = args
+        return this.messaging.request(name, params)
+    }
+
```

Comment:
nobody will ever call these un-typed - they are here purely to proxy the params through to the messaging library.

This means features authors no longer do

```js
this.messaging.request("abc", {def: ""})
```

instead, they just call this directly, which will only allow messages which have been defined via the schema
```
this.request("abc", {def: ""})
```